tools/webpack/packages.js

```diff
@@ -149,10 +149,8 @@ module.exports = function( env = { environment: 'production', watch: false, buil
 		},
 		plugins: [
 			new DefinePlugin( {
-				// Inject the `GUTENBERG_PHASE` global, used for feature flagging.
-				'process.env.GUTENBERG_PHASE': 1,
-				// Inject the `COMPONENT_SYSTEM_PHASE` global, used for controlling Component System roll-out.
-				'process.env.COMPONENT_SYSTEM_PHASE': 0,
+				// Inject the `IS_GUTENBERG_PLUGIN` global, used for feature flagging.
+				'process.env.IS_GUTENBERG_PLUGIN': 1,
```

Comment:
This relfects changes applied in the Gutenberg plugin.

Reply:
Actually, this should be set to `false`. We should exclude all changes from the Guetenberg plugin that are experimental.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/2564/commits/735f98785412c02ca3fd7c1204e6908264d16375